sdk/KoreBuild/modules/noop/module.targets

```diff
@@ -0,0 +1,10 @@
+<!-- Don't do anything. Useful for scripts that need to initialize KoreBuild without building anything. -->
+<Project>
+
+  <Target Name="Noop" />
+
+  <Target Name="Cow" DependsOnTargets="Noop">
+    <Message Text="Moo!" Importance="High" />
```

Comment:
Should check for existence of [`cowsay`](https://en.wikipedia.org/wiki/Cowsay) and use it. :trollface: